Source/Core/Core/HLE/HLE_OS.cpp

```diff
@@ -225,7 +225,7 @@ std::string GetStringVA(u32 str_reg, ParameterType parameter_type)
       {
       case 's':
         result += StringFromFormat(ArgumentBuffer.c_str(),
-                                   PowerPC::HostGetString(ap->GetArgT<u32>()).c_str());
+                                   PowerPC::HostGetString(ap->GetArgT<u32>()).value_or("").c_str());
```

Comment:
While you're here, isn't this `c_str()` unnecessary in `StringFromFormat`?

Reply:
Seems to be required:
`warning C4840: non-portable use of class 'std::basic_string<char,std::char_traits<char>,std::allocator<char>>' as an argument to a variadic function`

Reply:
Ah sorry.  Thought this was one of the functions converted to fmt.  Guess not

Reply:
Oh.  Looking back through previous reviews, that must be a spot that hasn't been converted yet.  `StringFromFormat` was getting switched to `fmt::format` slowly.  I was remembering wrong and thought  that `StringFromFormat` itself was getting converted to use `fmt`.

Sorry about that.